### PR TITLE
More types

### PR DESCRIPTION
**Pull request by vladimir-kotikov**
### Changes

Adds typings for `helpers` and `errors` modules

### Checklist

- [x] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: aiokafka/helpers.py [hunk 1]
```diff
@@ -3,15 +3,23 @@
...
-                       certfile=None, keyfile=None, password=None,
-                       crlfile=None):
+def create_ssl_context(
+    *,
+    cafile: str = None,
+    capath: str = None,
+    cadata: str = None,
+    certfile: str = None,
+    keyfile: str = None,
+    password: str = None,
```
Comment (gnsiva) [inline aiokafka/helpers.py:20]: The typing of these should look like `cafile: Optional[str] = None`

File: setup.cfg [hunk 2]
```diff
@@ -18,31 +18,8 @@ known_first_party=aiokafka,kafka
...
-
-[mypy-kafka.vendor.*]
-ignore_missing_imports = True
-
-[mypy-aiokafka.*]
-ignore_missing_imports = True
-
-[mypy-aiokafka.util]
-ignore_missing_imports = False
+python_version = 3.7
```
Comment (gnsiva) [inline setup.cfg:21]: python 3.7 has already [been deprecated](https://github.com/aio-libs/aiokafka/releases/tag/v0.8.1) in the latest version

File: stubs/kafka/structs.pyi [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    host: str
+    port: int
+    rack: str
+
+class PartitionMetadata(NamedTuple):
+    topic: str
+    partition: int
+    leader: int
+    replicas: list[int]
+    isr: list[int]
```
Comment (gnsiva) [inline stubs/kafka/structs.pyi:19]: This form of typing is not available in Python 3.8 which is still supported.
It should be:

```python
from typing import List, NamedTuple

...

    replicas: List[int]
    isr: List[int]
```

File: stubs/kafka/errors.pyi [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+class PolicyViolationError(BrokerResponseError): ...
+class SecurityDisabledError(BrokerResponseError): ...
+class NonEmptyGroupError(BrokerResponseError): ...
+class GroupIdNotFoundError(BrokerResponseError): ...
+class KafkaUnavailableError(KafkaError): ...
+class KafkaTimeoutError(KafkaError): ...
+
+# class FailedPayloadsError(KafkaError):
+#     payload: Incomplete
+#     def __init__(self, payload, *args) -> None: ...
```
Comment (gnsiva) [inline stubs/kafka/errors.pyi:88]: cant this just be deleted if it is commented out?

**Comment by ods** [comment]
Superseded by #932, #988, #989

**Pull request closed without merging**